### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by punkratz312**

### CODE REVIEW

File: random-beans/src/main/java/io/github/benas/randombeans/randomizers/collection/MapRandomizer.java [hunk 1]
```diff
@@ -49,16 +49,6 @@
     private final Randomizer<V> valueRandomizer;
 
     /**
-     * Create a new {@link MapRandomizer} with a random number of entries.
-     *
-     * @param keyRandomizer   the randomizer for keys
-     * @param valueRandomizer the randomizer for values
-     */
-    public MapRandomizer(final Randomizer<K> keyRandomizer, final Randomizer<V> valueRandomizer) {
```
Comment (PascalSchumacher) [inline random-beans/src/main/java/io/github/benas/randombeans/randomizers/collection/MapRandomizer.java]: Why do you want to remove these randomizers constructors and static factory methods?

Comment (punkratz312) [inline random-beans/src/main/java/io/github/benas/randombeans/randomizers/collection/MapRandomizer.java]: I used my IDE to remove all the dead code - if this is used, what could likely be true, i would write a test for it and readd it

Comment (PascalSchumacher) [inline random-beans/src/main/java/io/github/benas/randombeans/randomizers/collection/MapRandomizer.java]: These randomizer constructors and static factory methods are part of the API random beans exposes to it's users. To remove them means to break user code. This should not be done without a good reason and even then should only be done for the next major version (4.0).

**Comment by benas** [comment]
Hi @punkratz312 

There are some duplicate commits from #134 . Can you please update your PR with the master branch? Please don't remove public APIs, you can add tests to improve coverage instead. Also, I think we can add lombok annotations as you did in your last PR instead of removing setters. What do you think?

Kind regards
Mahmoud

File: random-beans/src/test/java/io/github/benas/randombeans/CollectionPopulationTest.java [hunk 2]
```diff
@@ -231,11 +229,6 @@ public void compositeCollectionTypesShouldBeEmpty() {
     }
 
     @Test(expected = ObjectGenerationException.class)
-    public void synchronousQueueTypeMustBeRejected() {
```
Comment (benas) [inline random-beans/src/test/java/io/github/benas/randombeans/CollectionPopulationTest.java]: This is a functional test about unsupported `SynchronousQueue` type. Please keep it

File: random-beans/src/test/java/io/github/benas/randombeans/CollectionPopulationTest.java [hunk 3]
```diff
@@ -231,11 +229,6 @@ public void compositeCollectionTypesShouldBeEmpty() {
...
-        enhancedRandom.nextObject(SynchronousQueueBean.class);
-    }
-
-    @Test(expected = ObjectGenerationException.class)
```
Comment (benas) [inline random-beans/src/test/java/io/github/benas/randombeans/CollectionPopulationTest.java]: This is a functional test about unsupported `DelayedQueue` type. Please keep it

**Comment by benas** [comment]
Any update on this PR?

**Comment by punkratz312** [comment]
Yeah i will fix that this Weekend;-)

Von meinem iPhone gesendet

**Comment by benas** [comment]
Great. I'm preparing the next release and would like to include it.

Many thanks upfront

**Comment by punkratz312** [comment]
I just got fired, sorry I can not make it anymore

**Comment by benas** [comment]
Oh sorry. Don't worry. Hope everything is well

**Pull request closed without merging**